tests/integration/replication.tcl

```diff
@@ -1172,7 +1172,7 @@ test {replicaof right after disconnection} {
 } {} {external:skip}
 
 test {Kill rdb child process if its dumping RDB is not useful} {
-    start_server {tags {"repl"}} {
+    start_server {tags {"repl"} } {
```

Comment:
need a save ""?
```suggestion
    start_server {tags {"repl"}} {
```

Reply:
i guess i added it and then removed it but didn't completely revert my change.
this test only creates 10 small keys, so this config doesn't really matters.